lighthouse-cli/performance-experiment/report/injected-script.js

```diff
@@ -30,17 +30,39 @@ window.addEventListener('DOMContentLoaded', _ => {
   rerunButton.style.display = 'inline-block';
 
   rerunButton.addEventListener('click', () => {
-    rerunButton.setAttribute('status', 'running');
-    rerunLighthouse().then(() => {
+    rerunButton.setAttribute('data-status', 'running');
+    rerunLighthouse({blockedUrlPatterns: getUrlPatternsToBlock()}).then(() => {
```

Comment:
changed to `getUrlBlockingConfig()` Because 'blocked' stands for URLs blocked in the previous run in the HTML.